src/components/Provider/Provider.tsx

```diff
@@ -92,15 +94,15 @@ class Provider extends React.Component<IProviderProps, any> {
 
     if (!fontFaces) return
 
-    const renderFontObject = font => {
+    const renderFontObject = (font: IFontFace) => {
       if (!_.isPlainObject(font)) {
         throw new Error(`fontFaces must be objects, got: ${typeof font}`)
       }
-      felaLtrRenderer.renderFont(font.name, font.path, font.style)
+      felaLtrRenderer.renderFont(font.name, font.paths, font.style)
```

Comment:
Fonts were also not being rendered because `path` was being used here instead of `paths`.  Added typings to ensure it doesn't happen again.